draft-ietf-taps-interface.md

```diff
@@ -1861,6 +1862,23 @@ lower-bound rate below which the application does not deem
 it will be useful. These are specified in bits per second. 
 The special value `Unlimited` indicates that no bound is specified.
 
+### Group Connection Limit
+
+Name:
+: groupConnLimit
+
+Type:
+: Numeric (with special value `Unlimited`)
+
+Default:
+: Unlimited
+
+This property controls the number of Connections that may arrive from
```

Comment:
Sorry - but being picky - you can't control what "may arrive", you can control what might be "accepted"

Reply:
Fixed; I wrote "...that can be accepted from ..."